src/app/containers/Timestamp/__snapshots__/index.test.jsx.snap

```diff
@@ -80,7 +80,7 @@ exports[`Timestamp should render correctly 1`] = `
     className="c1"
     dateTime="2018-10-19"
   >
-    19 October 2018
+    19 October 2018, 17:10 UTC
```

Comment:
Locally when running snapshots, I get the value: `18:10`. Not sure why. Would be good if others can reproduce this.

Reply:
I also get this, I guess jenkins is on UTC but locally we aren't? 🤷‍♂️